workshop/SCHEDULE.md

```diff
@@ -56,18 +55,18 @@ Minneapolis, MN 55414 in Room 105.
 |-------------|------------------------------------------------|
 | **Day 3**   | **2024-08-21 <br> Bulk RNA-Seq**  |
 | 8:30 AM     | Breakfast and coffee available |
-| 9:00 AM     | [RNA-Seq quantification (trim, filter, quant)](../completed-notebooks/RNA-seq/01-qc_trim_quant.nb.html) |
+| 9:00 AM     | Introduction to bulk RNA-Seq <br> [Introduction to bulk RNA-seq slides (PDF)](../slides/2024-08-20_intro-to-bulk-RNAseq.pdf) <br> [RNA-Seq quantification (trim, filter, quant)](../completed-notebooks/RNA-seq/01-qc_trim_quant.nb.html) |
```

Comment:
You would both have a slightly better sense than I would if this is enough time for both, so I'll let @jashapiro weigh in!

Reply:
(noting that your opening comment does note that not necessarily worth estimating this, which is good w/ me!)